frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/bucket-class-spec.ts

```diff
@@ -30,28 +27,77 @@ describe('Tests creation of Bucket Class', () => {
     cy.byTestID('item-create').click();
   });
 
+  afterEach(() => {
+    verifyBucketClass();
+    deleteBucketClass();
+  });
+
   after(() => {
-    cleanup();
+    config.cleanup();
     cy.logout();
   });
 
   it('Create a 1 Tier(Spread) Bucket Class', () => {
-    tiers = [Tier.SPREAD];
-    createAndTestBucketClass(tiers);
+    config.tiers = [Tier.SPREAD];
+    createBucketClass(config);
   });
 
   it('Create a 1 Tier(Mirror) Bucket Class', () => {
-    tiers = [Tier.MIRROR];
-    createAndTestBucketClass(tiers);
+    config.tiers = [Tier.MIRROR];
+    createBucketClass(config);
   });
 
   it('Create a 2 Tier(Spread, Spread) Bucket Class', () => {
-    tiers = [Tier.SPREAD, Tier.SPREAD];
-    createAndTestBucketClass(tiers);
+    config.tiers = [Tier.SPREAD, Tier.SPREAD];
+    createBucketClass(config);
   });
 
   it('Create a 2 Tier(Spread, Mirror) Bucket Class', () => {
-    tiers = [Tier.SPREAD, Tier.MIRROR];
-    createAndTestBucketClass(tiers);
+    config.tiers = [Tier.SPREAD, Tier.MIRROR];
+    createBucketClass(config);
+  });
+});
+
+describe('Tests creation of Namespace Bucket Class', () => {
+  const config = new NamespaceBucketClassConfig(
+    ['ns1', 'ns2', 'ns3', 'ns4'],
+    BucketClassType.NAMESPACE,
+  );
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    config.setup();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    commonFlows.navigateToOCS();
+    cy.byLegacyTestID('horizontal-link-Bucket Class').click();
+    cy.byTestID('item-create').click();
```

Comment:
These steps are repeating in both of the bucket class test suites.
Lets extract them as well into a single function

Reply:
ok